.github/workflows/test.yml

```diff
@@ -44,3 +44,58 @@ jobs:
           source activate          
           pip install pytest
           pytest
+
+  paths-filter:
+    runs-on: ubuntu-latest
+    outputs:
+      tag-file-modified: ${{ steps.filter.outputs.tag-file }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Path Filter
+        id: filter
+        uses: dorny/paths-filter@v3
+        with:
+          filters: |
+            tag-file:
+              - 'ersilia/hub/content/metadata/tag.txt'
+
+      - name: Tag file modified
+        if: steps.filter.outputs.tag-file == 'true'
+        run: echo "Tag file modified"
+
+      - name: Tag file not modified
+        if: steps.filter.outputs.tag-file != 'true'
+        run: echo "Tag file not modified"
+
+  update-model-request:
+    runs-on: ubuntu-latest
+    needs: paths-filter
+    if: needs.paths-filter.outputs.tag-file-modified == 'true'
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pyyaml
+
+      - name: Run update_model_request.py
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: python .github/scripts/update_model_request.py
```

Comment:
You either need to cd into the ersilia directory here or provide the relative path.